### DISCUSSION TITLE
mac m1使用cli打包网站时失败

### DISCUSSION THREAD

**Discussion by wilbur-yu** [tw93/Pake#285] [Q&A] [answered]
➜ pake https://acdtest.test.cn/admin
✔ We detected that you have not installed Rust. Install it now? … yes
⠇ Downloading Rustinfo: downloading installer
⠦ Downloading Rustinfo: profile set to 'default'
info: default host triple is aarch64-apple-darwin
info: syncing channel updates for 'stable-aarch64-apple-darwin'
⠧ Downloading Rustinfo: latest update on 2023-01-26, rust version 1.67.0 (fc594f156 2023-01-24)
info: downloading component 'cargo'
⠋ Downloading Rustinfo: downloading component 'clippy'
⠏ Downloading Rustinfo: downloading component 'rust-docs'
⠦ Downloading Rustinfo: downloading component 'rust-std'
⠴ Downloading Rustinfo: downloading component 'rustc'
⠸ Downloading Rustinfo: downloading component 'rustfmt'
⠏ Downloading Rustinfo: installing component 'cargo'
⠼ Downloading Rustinfo: installing component 'clippy'
⠧ Downloading Rustinfo: installing component 'rust-docs'
⠦ Downloading Rustinfo: installing component 'rust-std'
⠏ Downloading Rustinfo: installing component 'rustc'
⠋ Downloading Rustinfo: installing component 'rustfmt'
⠸ Downloading Rustinfo: default toolchain set to 'stable-aarch64-apple-darwin'

⠙ Downloading Rust  stable-aarch64-apple-darwin installed - rustc 1.67.0 (fc594f156 2023-01-24)

Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, run:
source "$HOME/.cargo/env"
✔ Downloading Rust
✔ please input your application name … burtonsys
You have not provided an app icon, use the default icon.(use --icon option to assign an icon)

added 81 packages, and audited 234 packages in 57s

77 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> pake-cli@1.2.4 build
> npm run tauri build --release

> pake-cli@1.2.4 tauri
> tauri build

thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/interface/rust.rs:762:69
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, copyfile '/opt/homebrew/lib/node_modules/pake-cli/src-tauri/target/release/bundle/dmg/test_1.0.0_aarch64.dmg' -> '/Users/wenbo/test.dmg'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'copyfile',
  path: '/opt/homebrew/lib/node_modules/pake-cli/src-tauri/target/release/bundle/dmg/test_1.0.0_aarch64.dmg',
  dest: '/Users/wenbo/test.dmg'
}

Node.js v19.4.0

target/release/bundle/dmg这个路径实际是不存在的.所以对应的dmg也是没有生成出来

**Comment by Tlntin** [comment, marked as answer]
可以尝试：
1. 关闭item终端再重新打开。
2. 或者将npm的模块安装到~目录，参考这个[链接](https://stackoverflow.com/questions/16151018/how-to-fix-npm-throwing-error-without-sudo)
3. 重装pake-cli
